core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractProject.java

```diff
@@ -90,29 +93,34 @@
   @GeneratedValue(strategy = GenerationType.AUTO)
   private long projectId = AbstractProject.UNSAVED_ID;
 
-  @Transient
+  @OneToMany(targetEntity = SampleImpl.class, fetch = FetchType.LAZY, mappedBy = "project")
   private Collection<Sample> samples = new HashSet<>();
-  @Transient
-  private Collection<Run> runs = new HashSet<>();
-  @Transient
+  @ManyToMany(targetEntity = StudyImpl.class, fetch = FetchType.LAZY)
+  @JoinTable(name = "Project_Study", joinColumns = { @JoinColumn(name = "studies_studyId") }, inverseJoinColumns = {
+      @JoinColumn(name = "Project_projectId") })
   private Collection<Study> studies = new HashSet<>();
-  @Transient
+  @OneToMany(targetEntity = ProjectOverview.class, mappedBy = "project")
```

Comment:
Should there be a CascadeType here? (if so, Dao code will need to be updated to remove the ProjectOverview from the project before ProjectOverview deletion)